recipes/openssl/1.x.x/conanfile.py

```diff
@@ -779,10 +780,20 @@ def _patch_install_name(self):
             makefile = "Makefile" if self._full_version >= "1.1.1" else "Makefile.shared"
             tools.replace_in_file(makefile, old_str, new_str, strict=self.in_local_cache)
 
+    @property
+    def _runtime(self):
+        if self.settings.compiler == "msvc":
+            return "M{}{}".format(
+                    "T" if self.settings.compiler.runtime == "static" else "D",
+                    "d" if self.settings.compiler.runtime_type == "Debug" else "",
+                )
+        else:
+            return self.settings.compiler.runtime
```

Comment:
```suggestion
        if self.settings.compiler == "Visual Studio":
            return self.settings.compiler.runtime
        else:
            return "M{}{}".format(
                "T" if self.settings.compiler.runtime == "static" else "D",
                "d" if self.settings.compiler.runtime_type == "Debug" else "",
            )
```

Small improvement: it avoids to fail if a consumer has forgotten to update its `settings.yml`